pkgs/applications/blockchains/monero-gui/default.nix

```diff
@@ -1,27 +1,27 @@
-{ mkDerivation, lib, makeDesktopItem, fetchFromGitHub
+{ stdenv, wrapQtAppsHook, makeDesktopItem, fetchFromGitHub
```

Comment:
Is it necessary to pass `stdenv`? This package is imported with `libsForQt5.callPackage`, and its preferable to use the `mkDerivation` that it provides.

Reply:
I know, however it seems `mkDerivation` will be no longer necessary/remove in the future... #71089.

Reply:
Ok, good to know, thanks.